### ISSUE TITLE
[1.14.4] Bedrolls and Sleeping Bags

### ISSUE THREAD

**Issue by frakier** [Quetzi/Morpheus#73]
Morpheus-1.14.4-4.2.44.jar
Option to prevent Bedrolls and Sleeping Bags from setting spawn points please.

**Comment by Quetzi** [comment]
I specifically look for the vanilla bed.

**Comment by frakier** [comment]
I have the mod only on the serverside, so I stopped the server and deleted the mod. 
Started the server [it is at my brothers house] client, logged in and no problem.
I have quite a few other mods installed maybe one is changing something about the Bedrolls, Sleeping Bags and Beds. I get a chance and I will run a server with just comforts and morpheus and see how it plays out.

**Comment by frakier** [comment]
Figured it out... 

[using the mods]
forge-1.14.4-28.1.107-installer.jar
comforts-FORGE-1.14.4-2.0-beta4.jar
Morpheus-1.14.4-4.2.44.jar
Upgrade+Aquatic+Wave+Three+v6+-+Beta+1.0.8.jar

Place a bed and set a spawn point...
Place bedroll or comforts and set a spawn point...
Kill self and you return to the bed.
Despite the "New spawnpoint has been set!" message ever time you click a bedroll or sleeping bag only the bed counts as a spawnpoint.  Note clicking the Bed again "You can sleep only at night and during thunderstoms". 

Without Morpheus installed...
Bed "You can sleep only at night and during thunderstoms".
Sleeping Bag "You can sleep only at night and during thunderstoms".
Bedroll "You can sleep only at night and during thunderstoms".

So to sum up the message "New spawnpoint has been set!" means nothing on the bedroll or the sleeping bag and can be ignored, the last bed you clicked is all that matters. Hope this helps.
